pkgs/games/tuxtype/default.nix

```
@@ -0,0 +1,39 @@
+{ stdenv, fetchurl, pkgconfig, librsvg, SDL, SDL_image, SDL_mixer, SDL_ttf }:
+
+stdenv.mkDerivation rec {
+  version = "1.8.3";
+  name = "tuxtype-${version}";
```

Nowadays, it is possible to use

```suggestion
  pname = "tuxtype";
```

`name = "${pname}-${version}"` will be created automatically.

---

Thanks